pkg/archive/archive_unix_test.go

```diff
@@ -248,9 +250,16 @@ func TestTarUntarWithXattr(t *testing.T) {
 		if len(changes) != 1 || changes[0].Path != "/3" {
 			t.Fatalf("Unexpected differences after tarUntar: %v", changes)
 		}
-		capability, _ := system.Lgetxattr(filepath.Join(origin, "2"), "security.capability")
-		if capability == nil && capability[0] != 0x00 {
-			t.Fatalf("Untar should have kept the 'security.capability' xattr.")
+
+		capability, err := system.Lgetxattr(filepath.Join(origin, "2"), "security.capability")
+		require.NoError(t, err)
+		assert.Equal(t, len(encoded), len(capability))
```

Comment:
`assert.Equal(t, encoded[:], capability)` maybe?

Or, to avoid duplication, `encoded := []byte…`.

(I didn’t test either, I might be missing something.)

Reply:
I didn't know I could do it that way. Thanks